lib/connector.js

```diff
@@ -271,6 +271,7 @@ Connector.defineAliases = function(cls, methodOrPropertyName, aliases) {
  */
 Connector.defineAliases(Connector.prototype, 'execute', ['command', 'query']);
 
+Connector.defineAliases(Connector.prototype, '_buildUpdate', 'buildUpdate');
```

Comment:
Now I have added `buildUpdate` as an alias of `_buildUpdate`; but I'm not too sure how to have a deprecation notice...